index.html

```diff
@@ -4,13 +4,152 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Global Enviromental Forum</title>
   <link rel="preconnect" href="https://fonts.googleapis.com">
   <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
   <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap" rel="stylesheet">
-  <link src="style.css" ref="stylesheet">
+  <link href="style.css" rel="stylesheet">
+  <script src="https://kit.fontawesome.com/db2c8620f4.js" crossorigin="anonymous"></script>
+  
 </head>
 <body>
-  <script src="index.js"></script>    
+  <header id="header-container">
+    <nav class="nav-bar">
+      <div class="mobile"><img class="hamburguer-menu" src="./Images/Hamburguer_Menu.png"/></div>
+      <a href="index.html"><img class="forum-logo" src="./Images/Forum Logo.png"/></a>
+      <ul class="navbar-menu-list desktop">
+        <li><a class="nav-bar-menu-li" href="about.html">About</a></li>
+        <li><a class="nav-bar-menu-li" href="#program">Program</a></li>
+        <li><a class="nav-bar-menu-li"href="#">Join</a></li>
+        <li><a class="nav-bar-menu-li">Sponsor</a></li>
+        <li><a class="nav-bar-menu-li" href="#">News</a></li>
+        <li><a class="gef-campaing" href="#">GEF Campaign</a></li>
+      </ul>
+    </nav>
+  </header>
+
+  <main>
+    <section class="home-intro">
+      <div class="home-data-container">
+        <h2>"Protect your future enviroment!!"</h2>
+        <h1>CONSERVATION GLOBAL FORUM 2023</h1>
+      <div>
+        <p class="home-intro-parag-container">A massive kickoff where the most respectable Sustainability Desginers will define a Global plan to implement the use of biggest companies 5% of its pre-tax profits to protect our nature.</p>
+      </div>  
+      <div class="home-intro-location-container">
+        <h2>2023.05.24(WED) ~ 26(FRI)</h2>
+        <p>@ Chapelco Golf Resort, San Martín de Los Andes, Argentine Patagonia. </p>
+      </div>
+    </div>
+    </section>
```

Comment:
You have a pretty decent design 🥇🥇 Well done. Please consider implementing the following to get a design almost identical to the given design guidelines 👇

- [ ] Add a background image for both pages. Please make sure that it look almost identical to the given design ✨
- [ ] Add some space/padding/margin between header and hero section following design guidelines
- [ ] Add a background color to `p` and reduce `font-size` a bit
- [ ] Increase the `font-size` and reduce the `font-weight` of _"Protect your future enviroment!!"_
- [ ] Increase the `font-size` of `.home-intro-location-container p` to `16px`


### **_Desktop version 👇_**

Your design | Given design 
---| ---|
![image](https://github.com/HFG43/Capstone1/assets/106119105/917ae657-8fb6-4793-b303-7d8fa184757f) | ![image](https://github.com/HFG43/Capstone1/assets/106119105/a51d894c-9d21-4ad2-b31b-01a87d26c24a)